internal/pkg/utils/errors/example_test.go

```diff
@@ -38,7 +39,10 @@ func ExampleWrap() {
 func ExampleWithStack() {
 	originalErr := errors.New("original error")
 	err := errors.WithStack(originalErr)
-	fmt.Println(errors.Format(err, errors.FormatWithStack()))
+	re := regexp.MustCompile(`\[.*/internal`)
+	fmt.Println(string(re.ReplaceAll([]byte(errors.Format(err, errors.FormatWithStack())), []byte("["))))
+	// output:
+	// original error [/pkg/utils/errors/example_test.go:40]
 }
```

Comment:
Examples without tests also make sense, for documentation.

Is it possible to define the output for each example?